hugo/content/about.md

```diff
@@ -0,0 +1,73 @@
+---
+title: About
+toc: true
+---
+
+# About the project
+
+Mumble is a free, open source, low latency, high quality voice chat application.
+
+## Source Code & Development
+
+You find the Source Code and further instructions regarding Building and Development in our [GitHub Repo](https://github.com/mumble-voip).
+
+## Licenses
+
+<!-- https://github.com/mumble-voip/mumble/blob/master/LICENSE -->
+
+<!-- more licenses? should be as correct and complete as possible -->
+
+# Who we are
+
+<!-- Title could also be: Developers, Team etc. -->
+
+<!-- improve -->
+
+We are a team of voluntary developers - both professional and non-professional coders - from different countries - including Germany, Italy, USA and more.
+
+<!--
+Link to Github Team page:
+https://github.com/orgs/mumble-voip/people
+
+-->
+
+<!-- Distinction between active and non-active developers? -->
+
+<!-- could contain more blabla about vision, free-software dedication etc. -->
+
+
+<!-- Credits/List of Contributors ? -->
+
+<!-- https://www.mumble.info/AUTHORS -->
```

Comment:
I would not link to that since it is probably outdated and furthermore it is exposing their mail addresses. Linking that on a website will just attract web crawlers.
Besides: All contributors can be found in the GitHub stats (or `git log` if someone really wants to know it)

Reply:
Ok, will use the stats/graph then.